src/routes/write-exercise/WriteExerciseScreen.tsx

```diff
@@ -23,6 +24,7 @@ const WriteExerciseScreen = ({ navigation, route }: WriteExerciseScreenPropsType
   const { id } = discipline
   const [currentDocumentNumber, setCurrentDocumentNumber] = useState(0)
   const [newDocuments, setNewDocuments] = useState<DocumentType[] | null>(null)
+  const { height } = useWindowDimensions()
```

Comment:
Normally we use the `react-native-responsive-screen` library to get window-dimension. See it makes sense to use `useWindowDimenstions()` here, but not sure though because it is somehow "inconsistent". 
What do you think?
(@klinzo)

Reply:
No idea about `react-native-responsive-screen` tbh, never worked with it yet.

Reply:
i added a custom hook for getting screen size without navbar and statusbar. pls check